core/src/main/java/com/bettercloud/bigtable/orm/BigTableEntityDao.java

```diff
@@ -311,4 +322,43 @@
 
         table.delete(deletes);
     }
+
+    private T delegateToEntity(final Result result) throws IOException {
+        final T entity = entityFactory.get();
+
+        final EntityConfiguration.EntityDelegate<T> delegate = delegateFactory.apply(entity);
+
+        for (final Column column : columns) {
+            final byte[] family = Bytes.toBytes(column.getFamily());
+            final byte[] qualifier = Bytes.toBytes(column.getQualifier());
+
+            final Cell cell = result.getColumnLatestCell(family, qualifier);
+
+            final Object value;
+
+            if (cell != null) {
+                final byte[] bytes = cell.getValueArray();
+
+                if (bytes.length > 0) {
+                    value = objectMapper.readValue(bytes, column.getTypeReference());
```

Comment:
This is a dangerous conversion for table scans. In the event a given column can contain multiple types, this will very likely throw an exception. If left as is, the entire scan will throw an exception and nothing will be returned. Alternatively, you can try/catch the conversion for every single column, but then you may have missing results. It's really up to you guys how you want to handle it. See the previously closed PR for a little more of an in-depth response.

Reply:
Ahh, gotcha. I'll see what I can do alternatively.

Reply:
Let's talk about this, I'm fine with the implementation of this logic, but I also agree that there is a better place that we can put this.